jobs/workers.ts

```diff
@@ -18,14 +15,17 @@ const syncAndSubscribeAddressList = async (addressList: Address[]): Promise<void
     })
   )
   // subscribe addresses
+  // WIP: this will be refactored in PR 411-6
```

Comment:
I don't think that is a good idea, because the transaction passed here will be the grpc transaction and outside the grpc or chronik files no data type of those blockchains can be used or known. This here would be knowing and treating the grpc Transaction parameter. In the PR 411-6 where I refactored the subscribeTransaction, all this logic goes inside the grpc and chronik files. Otherwise the worker.ts has to have an `if` to know which blockchain each address belongs to so that it can use its proper grpc or chronik Transaction types and methods.

Reply:
Fair enough, good sir, done